src/os/bluestore/BlockDevice.h

```diff
@@ -92,10 +92,10 @@ class BlockDevice {
   uint64_t size;
   uint64_t block_size;
   bool rotational = true;
-  aio_callback_t aio_callback;
-  void *aio_callback_priv;
 
 public:
+  aio_callback_t aio_callback;
+  void *aio_callback_priv;
```

Comment:
I think keep aio_callback and aio_callback_priv as protected, and set Task as friend is a better choice.